src/gui/gui-buffer.c

```diff
@@ -4421,6 +4427,7 @@ gui_buffer_print_log ()
         log_printf ("  text_search_regex_compiled: 0x%lx", ptr_buffer->text_search_regex_compiled);
         log_printf ("  text_search_where . . . : %d",    ptr_buffer->text_search_where);
         log_printf ("  text_search_found . . . : %d",    ptr_buffer->text_search_found);
+        log_printf ("  text_search_start_line: 0x%lx", ptr_buffer->text_search_start_line);
```

Comment:
Please align the value with other variables above and below (anyway, this variable will move to gui-window.c).